css/index.css

```diff
@@ -98,3 +98,36 @@ header a:not(:last-child) {
     align-items: center;
 }
 
+.img-section-2 {
+    background-image: url(/Users/maksim/Desktop/project/images/photo%20secondary%20section.png);
+    background-repeat: no-repeat;
+    background-position: right top;
+}
+
+.container-section-2 {

```

Comment:
.container shoulb be div inside every section. now you have bug, when bg image does not cover all section
![Screenshot from 2024-02-21 13-06-41](https://github.com/Maksim-Serov/test2/assets/53647180/9250bfdd-cfbb-4cdb-9d24-f2b4d843e23d)